renovate.json

```diff
@@ -1,5 +1,11 @@
 {
   "extends": [
     "config:base"
-  ]
+  ],
+  "pin": false,
```

Comment:
This is invalid config. I suggest you change Renovate config in a dedicated config-only PR that can be validated by the bot. All open PRs will be updated once you commit the config change to master.